src/plugins/intel_cpu/tests/functional/single_layer_tests/interpolate.cpp

```diff
@@ -334,12 +348,18 @@ const std::vector<fusingSpecificParams> interpolateFusingParamsSet{
         fusingFakeQuantizePerTensorRelu,
 };
 
-std::vector<std::map<std::string, std::string>> filterAdditionalConfig() {
+std::vector<std::map<std::string, std::string>> filterAdditionalConfig(bool enforceBF16 = true) {
     if (InferenceEngine::with_cpu_x86_avx512f()) {
+        if (enforceBF16) {
```

Comment:
look like to better use these configs in instansiation only for bf16

Reply:
Unfortunately, I haven't fully got your point.
The following key: `{InferenceEngine::PluginConfigParams::KEY_ENFORCE_BF16, InferenceEngine::PluginConfigParams::YES}` was present in original tests, and the only original precision used there was `ElementType::f32`
Do you suggest to completely remove this key until we add full BF16 support and corresponding tests?

Reply:
better to separate test instantiations for bf16 to standalone instance according using bf16 configs.

Reply:
Created separate function `filterAdditionalConfigNearestExt()` for Interpolate Nearest tests.
@iefode , could you please take a look?